string topic
string node_uri
---
string[] publisher_uris
